components/script/body.rs

```diff
@@ -104,6 +112,9 @@ fn run_package_data_algorithm<T: BodyOperations + DomObject>(object: &T,
         BodyType::Json => run_json_data_algorithm(cx, bytes),
         BodyType::Blob => run_blob_data_algorithm(&global, bytes, mime),
         BodyType::FormData => run_form_data_algorithm(&global, bytes, mime),
+        BodyType::ArrayBuffer => unsafe {
+            run_array_buffer_data_algorithm(cx, bytes)
```

Comment:
now `run_array_buffer_data_algorithm` is declared as unsafe fn, and this match arm uses unsafe scope.